### ISSUE TITLE
unable to upload .ico file for the favicon icon.

### ISSUE THREAD

**Issue by pankajbhope** [magento/magento2#9070]
Hello Team,

I am unable to upload the .ico file from admin, It is throwing an error.

error : We don't recognize or support this file extension type.

It seems like the ui-component is for the uploader is not configured properly.

Here is the code snippet.

```
        <field name="head_shortcut_icon">
                <argument name="data" xsi:type="array">
                    <item name="config" xsi:type="array">
                        <item name="label" xsi:type="string" translate="true">Favicon Icon</item>
                        <item name="formElement" xsi:type="string">fileUploader</item>
                        <item name="componentType" xsi:type="string">fileUploader</item>
                        <item name="notice" xsi:type="string" translate="true">Allowed file types: ico, png, gif, jpg, jpeg, apng, svg. Not all browsers support all these formats!</item>
                        <item name="maxFileSize" xsi:type="number">2097152</item>

                        **<item name="allowedExtensions" xsi:type="string">jpg jpeg gif png svg</item>**

                        <item name="uploaderConfig" xsi:type="array">
                            <item name="url" xsi:type="string">theme/design_config_fileUploader/save</item>
                        </item>
                    </item>
                </argument>
            </field>
```

**Comment by piotrkwiecinski** [comment]
It's known issue. I was fixed in develop branch https://github.com/magento/magento2/pull/8901/files 
and on github there is workaround for 2.1:
https://github.com/springimport/magento2-module-favicon-bugfix

**Comment by col-be** [comment]
Issue exists in 2.1.5; easy way around it is to just use a png.

**Comment by korostii** [comment]
Actually it was fixed even earlier than the commit you referred to, here: c9c4882dea9544cb867d76c5d962ef5de6ccf122 (9 months ago!)
Still hasn't made its way into a release.

It clearly has an internal ticket attached MAGETWO-54175 , so.. uhm.. maybe, it just got lost? (Not exactly a question I expect an answer to, just trying to think of a reasonable excuse)
Maybe such things don't become a priority unless they're reported 10x times on github.
"Easy to workaround, no-one complains, let's just deal with it later"
Or does every one-line fix have to go through the whole pregnancy cycle? Unbelievable. The whole release process seems to be a bit lacking. Copyright year update in February, what a joke.
There should really be a way to fast-track minor non-breaking fixes that wouldn't involve the whole "we plan it for version 2.1.8 but don't have an exact timeline yet" routine.

**Comment by magento-engcom-team** [comment]
@pankajbhope, thank you for your report.
The fix for this issue is already available in the `develop` branch and upcoming `2.2.0` release.
